README.md

```diff
@@ -1,2 +1,36 @@
-# webpack-boilerplate
-A simple yet powerful webpack boilerplate, which can be used as a starting point in projects. Built by following the webpack official guides.
+# Project Name
+
+Todo List
+
+This is a website that was created as a Microverse project. The website is a todo list site that allows users to add and remove tasks. The Task can be moved around according to importance.
+
+## Built With
+
+- HTML
+- JavaScript
+- Webpack
+- CSS
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps. You can either download the zip file, or clone the repository from [here](https://github.com/the-catalystmc/awesome-books/tree/build-website). After you can open the index.html file to load the website locally.
+
```

Comment:
- [ ] Please give descriptive instructions on how to run the project locally. Because you are using webpack, you need to provide the following instructions once the repo is cloned.
- `npm install` to install the packages
- `npm run start` to run the server on local server